src/css/modal.styl

```diff
@@ -0,0 +1,59 @@
+.modal-form-trigger
```

Comment:
Is this class used by the tutorial? I doubt it since the child class is something I think is used in the current PFE header... Let's not move unnecessary styles into this.